File: ProyectosC#/class/Funcionario.cs
public class Funcionario : Usuario
{
    public Funcionario(int id, string nombre, string carnet)
        : base(id, nombre, "Funcionario", carnet)
    {
    }

    public void ReservarCubiculo(Cubiculo cubiculo)
    {
        cubiculo.Reservar(this);
    }
}

File: Funcionario.java

package parking;

public class Funcionario extends Usuario{
    
    private String cargo;
    
    public Funcionario(String cargo, int id, String nombre) {
        super(id, nombre,"Funcionario");
        this.cargo=cargo;
    }
    public void reservCubiculo(Cubiculos cube) {
        cube.Reservar(this);
    }
    
    
    
}
